session.go

```diff
@@ -451,12 +437,14 @@ func (s *Session) maybeLookupLocation(ctx context.Context) (err error) {
 		runtimex.PanicOnError(err, "s.lookupASN #1 failed")
 		cc, err = s.lookupProbeCC(s.CountryDatabasePath(), probeIP)
 		runtimex.PanicOnError(err, "s.lookupProbeCC failed")
-		resolverIP, err = s.lookupResolverIP(ctx)
-		runtimex.PanicOnError(err, "s.lookupResolverIP failed")
-		resolverASN, resolverOrg, err = s.lookupASN(
-			s.ASNDatabasePath(), resolverIP,
-		)
-		runtimex.PanicOnError(err, "s.lookupASN #2 failed")
+		if s.proxyURL == nil {
```

Comment:
If we're using a user enforced proxy, then it's not very useful to discover info about a resolver that we don't use and that may actually be misleading if the person looking into the data is not paying attention.